weekly_report/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM public.ecr.aws/lambda/python:3.12
-
-WORKDIR ${LAMBDA_TASK_ROOT}
-
+FROM python:latest
```

Comment:
Very good spot to change these features on the Dockerfile; something I forgot had to be done!